app/src/main/java/org/gdg/frisbee/android/chapter/ChapterAdapter.java

```diff
@@ -32,33 +29,12 @@
 
 public class ChapterAdapter extends ArrayAdapter<Chapter> {
 
-    private final boolean mWhiteText;
-    private LayoutInflater mInflater;
-
-    public ChapterAdapter(Context context, boolean whiteText) {
-        super(context, android.R.layout.simple_list_item_1);
-        mInflater = LayoutInflater.from(context);
-        mWhiteText = whiteText;
-    }
-
-    @Override
-    public View getView(final int position, final View convertView, final ViewGroup parent) {
-        if (!mWhiteText) {
-            return super.getView(position, convertView, parent);
-        } else {
-            View view = convertView;
-            if (view == null) {
-                view = mInflater.inflate(R.layout.spinner_item_actionbar, parent, false);
-            }
-            TextView textView = (TextView) view.findViewById(android.R.id.text1);
-            textView.setText(getItem(position).getShortName());
-            return view;
-        }
+    public ChapterAdapter(Context context, @LayoutRes int layoutRes) {
+        super(context, layoutRes);
     }
 
-
     @Override
-    public void addAll(Collection<? extends Chapter> chapters) {
+    public void addAll(@NonNull Collection<? extends Chapter> chapters) {
```

Comment:
Better only annotate @Nullable and assume everything else as non null. Or is it different here?